README.rst

```diff
@@ -236,7 +236,12 @@ Compatibility with GMT/Python/NumPy versions
       - >=6.3.0
       - >=3.8
       - >=1.19
-    * - `v0.6.1 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.6.1>`_ (latest release)
+    * - `v0.7.0 <https://github.com/GenericMappingTools/pygmt/releases/tag/v0.7.0>`_ (latest release)
+      - `v0.7.0 Documentation <https://www.pygmt.org/v0.7.0>`_
+      - >=6.3.0
+      - >=3.8
+      - >=1.19
```

Comment:
```suggestion
      - >=1.20
```

Reply:
Maybe put this change in #1985, following what was done in previous NEP29 PRs.

Reply:
Done, but we should leave this suggestion open because that is for the `dev` row and this is for the `v0.7.0` row.

Reply:
Ah yes, let's keep this here then.